python/ray/_private/metrics_agent.py

```diff
@@ -73,6 +75,41 @@ def name(self):
 Record = namedtuple("Record", ["gauge", "value", "tags"])
 
 
+def fix_metric(metric: Metric):
+    """
+    Fix the inbound `opencensus.proto.metrics.v1.Metric` protos to make it acceptable
+    by opencensus.stats.DistributionAggregationData.
+
+    - metric name: gRPC OpenCensus metrics have names with slashes and dots, e.g.
+    `grpc.io/client/server_latency`[1]. However Prometheus metric names only take
+    alphanums,underscores and colons[2]. We santinize the name by replacing non-alphanum
+    chars to underscore, like the official opencensus prometheus exporter[3].
+    - distribution bucket bounds: The Metric proto asks distribution bucket bounds to
+    be > 0[4]. However, gRPC OpenCensus metrics have their first bucket bound == 0 [1].
```

Comment:
```suggestion
    be > 0[4]. However, gRPC OpenCensus metrics have their first bucket bound == 0 [1].
```

What does [4] mean here?

Reply:
the fourth reference link.